app/build.gradle.kts

```diff
@@ -187,6 +195,8 @@ dependencies {
     androidTestImplementation(libs.ui.test.junit4)
     androidTestImplementation(libs.junit)
     androidTestImplementation(libs.compose.test.junit)
+    androidTestImplementation(libs.mockk.android)
+    androidTestImplementation("io.mockk:mockk-agent:1.13.12")
```

Comment:
Changes: this dependency contains a hardcoded version directly in `build.gradle.kts` which is not ideal, could you put the version in `libs.versions.toml` instead, like the other dependencies?